moduleroot/spec/spec_helper.rb.erb

```diff
@@ -31,6 +31,38 @@ default_fact_files.each do |f|
   end
 end
 
+# @return [String] - the path to the fixtures directory
+def fixtures_dir
```

Comment:
`puppetlabs_spec_helper` already provides this value as `fixture_path` but as a [local variable](https://github.com/puppetlabs/puppetlabs_spec_helper/blob/07978467ed536ce1fcc8b2d0dbb830800e06c9b9/lib/puppetlabs_spec_helper/module_spec_helper.rb#L15). Rather than duplicate this logic, we should change `puppetlabs_spec_helper` to expose this value.

Reply:
Seems reasonable.